modules/data-table/main/index.coffee

```diff
@@ -663,8 +663,6 @@ class DataTable extends hx.EventEmitter
           start = undefined
           end = undefined
 
-        selection.classed('hx-data-table-infinite', totalCount is undefined)
```

Comment:
allow for the table to change whether it is still considered infinite or not by checking for that at a later point